internal/stringutil/htmlchunker_test.go

```
@@ -0,0 +1,1505 @@
+		MaxTokens:      max,
+		MinTokens:      min,
+		OverlapTokens:  overlap,
+		TokenizerFunc:  simpleTokenizer,
+		PreserveBlocks: []string{"pre", "code", "table"},
+	}
+}
+
+func generateHTML(tag, content string, count int) string {
+	var b strings.Builder
+	for i := 0; i < count; i++ {
+		b.WriteString(fmt.Sprintf("<%s>%s %d</%s>\n", tag, content, i, tag))
```

golangci-lint (2.12.2)

[error] QF1012: Use fmt.Fprintf(...) instead of WriteString(fmt.Sprintf(...))

(staticcheck)